agents/holo_fpga/roach_agent.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import os
+import numpy as np
+import txaio
+import yaml
+import time
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    import casperfpga
+    from holog_daq import fpga_daq3, synth3
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock
+
+
+class FPGAAgent:
+    """
+    Agent for connecting to the Synths for holography
+
+    Args: $OCS_CONFIG_DIR holog_config.yaml
+    """
+
+    def __init__(self, agent, config_file):
+
+        self.fpga = None
+        self.initialized = False
+        # self.take_data = False
+
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+
+        """
+        ### register the position feeds
+
+        ### Agent registers data feed, things published to feed go to grafana and .g3 files
+        ### pick a good name for feeds, once it's registered it's kinda permanent
+
+        """
```

Comment:
Having these comments is fine, but putting them in triple quotes is making them show up as documentation for `self.lock`. Either drop altogether or convert them to leading single `#` only with no wrapping `"""`.

Reply:
ok i think this was left over from an example agent so im just dropping them.